base/hacks/load_kilda_dev_ubuntu.sh

```diff
@@ -85,7 +85,7 @@ echo ##  PYTHON ETC
 echo ##########################
 
 sudo apt-get update
-sudo apt-get install -y apt-utils python python-pip
```

Comment:
Where is it(this whole file) used?

Reply:
This file uses for prepare local dev environment. It installs all necessary stuff, like java, python, compose, etc...

Reply:
i.e. it should be started manually to prepare local env(linux ubuntu) to build open-kilda?

Can't find its mentions in docs/wiki (actually I can't find its mentions anywhere).